content/en/docs/contributing/talk-to-us.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Meet the maintainers"
+linkTitle: "talk-to-us"
+weight: 70
+type: "docs"
+---
+
+## Meetings
+
+All cert-manager meetings are open for everyone to join.
+
+To get invites you can subscribe to [our mailing list](https://groups.google.com/forum/#!forum/cert-manager-dev).
+
+We have 2 regular repeating meetings:
+
+* daily stand-up meetings [on Google Meet](https://meet.google.com/eum-fyvt-xpa) at [10:30 London time](http://www.thetimezoneconverter.com/?t=10:30&tz=Europe/London) every weekday
+* bi-weekly developer meetings [on Google Meet](https://meet.google.com/abp-bwhk-wxc) at [17:00 London time](http://www.thetimezoneconverter.com/?t=17:00&tz=Europe/London) (for dates, check calendar invites or [meeting notes](https://docs.google.com/document/d/1Tc5t6ylY9dhXAan1OjOoldeaoys1Yh4Ir710ATfBa5U))
```

Comment:
Suggestion: note the fact that these meetings are recorded and provide a link to our YT page